src/OneExplorer/ConfigObject.ts

```diff
@@ -112,12 +121,31 @@ export class ConfigObj {
     return found ? true : false;
   }
 
+  /**
+   * @brief getter for config setting
+   */
+  public get configSetting(): ConfigSetting {
+    let configSetting: ConfigSetting;
+    switch (this.configType) {
+      case CfgType.one:
+      default:
+        configSetting = new OneConfigSetting();
```

Comment:
(Q) Does it support multiple config setting? Is this `configSetting` shared on all backends?

Reply:
> Does it support multiple config setting?

The answer first, is : Yes.

Here, it represents the configuration type of a file. So each file can have one config setting. Here, *.cfg corresponds to 
OneConfigSetting and *.edgetpucfg corresponds to EdgeTPUConfigSetting (in the following PR).